bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2018.html

```diff
@@ -0,0 +1,257 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+  {% extends "mozorg/about/policy/transparency/report-base.html" %}
+
+  {% block page_title %}{{ _('Transparency Report - January-June, 2018') }}{% endblock %}
+
+  {% block report_title %}
+    <h1>{{ _('Transparency Report') }} <span>{{ _('Reporting Period: January 1, 2018 to June 30, 2018') }}</span></h1>
+  {% endblock %}
+
+  {% block report_body %}
+    <section class="section" id="government-user-data">
+      <h2 class="section-title" data-accordion-role="tab">{{ _('Government Demands for User Data') }}</h2>
+
+      <div class="content" data-accordion-role="tabpanel">
+        <p>
+          {{ _('In the Reporting Period, Mozilla received 1 Court Order and 1 Subpoena for user data.') }}
+        </p>
+
+        <table class="data-table">
+          <thead>
+            <tr>
+              <th scope="col">{{ _('Legal Processes') }}</th>
+              <th scope="col">{{ _('Received') }}</th>
+              <th scope="col">{{ _('Data Produced') }}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-search-warrant' }}">{{ _('Search Warrants') }}</a>
+              </th>
+              <td>0</td>
+              <td>{{ _('N/A') }}</td>
+            </tr>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-subpoena' }}">{{ _('Subpoenas') }}</a>
+              </th>
+              <td>1<sup><a href="#footnote-1">2</a></sup></td>
```

Comment:
Need to update these links for the separate footnote IDs, so 1 links to `#footnote-1`, 2 links to `#footnote-2`, etc. 

And should also rearrange them in the right numerical order, so 1 for subpoenas, 2 for court orders, 3 for national security requests.